src/common/crypto/mbedtls/cryptoprovider.cpp

```diff
@@ -269,6 +270,25 @@ Error MbedTLSCryptoProvider::PEMToX509Certs(const String& pemBlob, Array<x509::C
     return ErrorEnum::eNone;
 }
 
+#define PEM_BEGIN_CRT "-----BEGIN CERTIFICATE-----\n"
```

Comment:
Do  not use define. `static constexpr auto` as private or probably public member.

Reply:
I will change. These consts were copy-pasted as is from mbedtls

Reply:
Why don't use them from mbedtls? Are they public in mbedTLS?